src/server/set_family.cc

```diff
@@ -98,6 +101,14 @@ struct StringSetWrapper {
       ss->Reserve(entries_len);
     }
     for (string_view member : EntriesRange(entries)) {
+      // Overwrite TTL for existing members. This also makes res not count for existing members.
+      if (update_ttl_for_existing) {
+        if (auto it = ss->Find(member); it != ss->end()) {
+          it.SetExpiryTime(ttl_sec);
+          continue;
+        }
+      }
+
```

Comment:
Another option to do this may be to set this TTL in `StringSet::AddBatch` when `FindInternal` finds an existing field.

Reply:
changed to set this in `AddBatch` as otherwise we have to do two lookups for the same key. It does require wiring the bool parameter down to `AddBatch`